packages/vuepress-theme-odyssey/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@okta/vuepress-theme-odyssey",
+  "version": "0.6.0",
+  "description": "VuePress theme for Odyssey Documentation",
```

Comment:
Similar to our other package, let's make it `private`.

Reply:
Fixed in #728